src/main/kotlin/no/nav/personbruker/dittnav/eventaggregator/beskjed/BeskjedEventService.kt

```diff
@@ -42,7 +42,9 @@ class BeskjedEventService(
                 } catch (fve: FieldValidationException) {
                     countFailedEventForProducer(event.systembruker)
                     val eventId = event.getNonNullKey().getEventId()
-                    log.warn("Klarte ikke transformere eventet pga en valideringsfeil. EventId: $eventId, context: ${fve.context}", fve)
+                    val systembruker = event.getNonNullKey().getSystembruker()
+                    val msg = "Klarte ikke transformere eventet pga en valideringsfeil. EventId: $eventId, systembruker: $systembruker"
+                    log.warn(msg, fve.toString(), fve)
```

Comment:
Kommer egentlig `fve.toString()` i selve logginnslaget? Må det ikke være med en `{}` for å angi at dette skal komme med? Jeg mener at det bare er det siste argumentet, selve exception-en, som ikke må ha en egen `{}` for å komme med i logginnslaget.